# Key value pair for sts_behavior_tree color updates
# Event enum: 0 = boxColor; 1 = fontColor
uint16 event #what kind of change
string target #object of interest
string attribute #which attribute to change
string value #value to which to change
